javascript/builtins/Error.json

```diff
@@ -228,6 +228,56 @@
             }
           }
         },
+        "captureStackTrace": {
+          "__compat": {
+            "tags": [
+              "web-features:javascript"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": "17"
+              },
+              "chrome_android": "mirror",
+              "deno": {
+                "version_added": false
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "136",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "javascript.options.experimental.error_capture_stack_trace",
+                    "value_to_set": "true"
+                  }
+                ],
+                "impl_url": "https://bugzil.la/1886820"
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": "16.17.0"
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "17.3"
```

Comment:
Does not show up in 16.5 - so 17.3 is the first version I found it in. Wasn't allowed to use a ranged value.

Reply:
17.3 seems correct per https://github.com/WebKit/WebKit/commit/997e074bb35ed07b69c9b821141c91dd548e0d02